doc/ref_model/chapters/chapter03.md

```diff
@@ -13,21 +12,21 @@
 * [3.4 Physical Resources](#3.4)
 * [3.5 Network](#3.5)
 
-There is the necessity to clearly define which kind of infrastructure resources a shared network function virtualisation infrastructure (NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNF), so that the requirements of the workloads match the capabilities of the NFVI.
+There is the necessity to clearly define which kind of infrastructure resources a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNF), so that the requirements of the workloads match the capabilities of the cloud infrastructure.
 
-The lack of a common understanding of which resources and corresponding capabilities a suitable NFVI should provide may lead to several issues which could negatively impact the time and the cost for on-boarding and maintaining these solutions on top of a virtualised infrastructure. For Example:
+The lack of a common understanding of which resources and corresponding capabilities a suitable cloud infrastructure should provide may lead to several issues which could negatively impact the time and the cost for on-boarding and maintaining these solutions on top of a virtualised infrastructure. For Example:
 
-- Supporting any kind of workload specific requirements (e.g. regarding network acceleration or API access) might result in having to establish different silo of NFVIs for each workload type.
+- Supporting any kind of workload specific requirements (e.g. regarding network acceleration or API access) might result in having to establish different silo of cloud infrastructures for each workload type.
 - Synchronising the release cycles of a large set of different technologies will sooner or later lead to situations in which required upgrades cannot be applied easily due to incompatibilities.
 
-The abstraction model presented in this chapter specifies a common set of virtual infrastructure resources which NFVI will need to provide to be able to host most of the typical VNF/CNF workloads required by the operator community.
+The abstraction model presented in this chapter specifies a common set of virtual infrastructure resources which a cloud infrastructure will need to provide to be able to host most of the typical VNF/CNF workloads required by the operator community.
 
 Although a couple of explicit and implicit abstraction models (e.g. in the context of ETSI NFV) are already available, they fall short when addressing the following design principles:
```

Comment:
DO we need a reference to these models that fall short?

Reply:
@pgoyal01, I have removed this sentence as being to contentious and unnecessary.  I do not think that we need to justify our efforts any more...